.github/workflows/schedule.py

```diff
@@ -148,13 +148,12 @@ def _is_eligible(cls, start: str, period: str, now: datetime):
 
     def create_issue(self, title_date: date) -> None:
         title = self.properties['title'] + ' ' + str(title_date)
-        flags = []
-        repository = self.properties.get('_repository')
-        if repository is not None:
-            flags.append(f'--repo={repository}')
+        repo = self.properties.get('_repository', [])
+        if repo:
+            repo = [f'--repo={repo}']
```

Comment:
I don't understand the default. What's wrong with the original approach? It should be an error to specify a front-matter entry without a value.